src/components/ProfileUpdate/index.jsx

```diff
@@ -71,13 +58,10 @@ const ProfileUpdate = (props) => {
       });
   };
   useEffect(() => {
-    const keys = ['firstName', 'lastName', 'major', 'bio'];
+    const keys = ['firstName', 'lastName', 'major', 'bio', 'graduationYear'];
     keys.forEach((key) => {
-      setFieldValue(key, user.profile[key]);
+      props.setFieldValue(key, props.user.profile[key]);
```

Comment:
For props you can just use the destructuring above for and replace ```props.setFieldValue``` with ```setFieldValue```. You also don't need to call props for ```props.user.profile``` because it already destructured in ```const { handleBlur, handleChange, handleSubmit, user, values } = props;```

Reply:
Resolved in 1fdcbdf.